site/app/templates/grading/electronic/Status.twig

```diff
@@ -561,6 +565,8 @@
                     Grade Inquiries: {{ GradeInq }}
                     {{ "\n"|nl2br }}
                     Incomplete Grading: {{ IncompGrading }}
+                    {{ "\n"|nl2br }}
```

Comment:
Go through this file and replace all instances of `{{ "\n"|nl2br }}` with `<br/>`. No reason to wait the cycles with this function when we 100% know the outcome anyway.